setup.py

```diff
@@ -291,10 +310,19 @@ def run(self):
     elif sys.platform == 'darwin':
         ext_modules.append(Extension('borg.platform_darwin', [platform_darwin_source]))
 
+
+def parse(root, describe_command=None):
+    return subprocess.check_output("git describe --tags --long").decode().strip()
+
+parsefunction = None
+if sys.platform == 'win32':
+    parsefunction = parse
```

Comment:
shorter and still readable:

parse_function = parse if sys.platform == 'win32' else None